kobo-uncaged/main.go

```diff
@@ -363,7 +363,8 @@ func (ku *KoboUncaged) readMDfile() error {
 	// make a temporary map for easy searching later
 	tmpMap := make(map[string]int, len(koboMD))
 	for n, md := range koboMD {
-		contentID := lpathToContentID(md.Lpath, string(ku.contentIDprefix))
+		newLpath := lpathKepubConvert(md.Lpath)
+		contentID := lpathToContentID(newLpath, string(ku.contentIDprefix))
```

Comment:
It's a bit cleaner and more readable not to have that extra intermediate variable.

Reply:
From a *what on earth was I thinking* standpoint, I tend not to be a fan of trying to do too much in a single line. Although I agree in this case that it's clear enough.